src-built-in/components/assets/css/_globals.css

```diff
@@ -1,3 +1,3 @@
 @import url('./_variables.css');
-@import url('./_variablesWhitelabel.css');
+@import url('./_variablesWhiteLabel.css');
```

Comment:
@DanNicolai since customers have already started to use Whitelabel (lowercase) let's just live with it. I don't want to have to step JPM through changing their capitalization.